src/Nethermind/Nethermind.Evm/VirtualMachine.cs

```diff
@@ -881,7 +876,7 @@ static void UpdateCurrentState(EvmState state, int pc, long gas, int stackHead)
                             BytesMax32.AsSpan(0, position).CopyTo(b[..position]);
                         }
 
-                        stack.PushBytes(b);
+                        stack.PushWord256(b);
```

Comment:
Can above if-esle be done with Vector256? I think so, we need to copy part and 0 other bytes?